<?xml version="1.0"?>

<launch>

	<rosparam command="load" file="$(find crazyflie_human)/config/safety_pred.yaml" />

	<!-- Human ID. -->
	<arg name="human_number" default="1" />
	<param name="human_number" value="$(arg human_number)" />

	<!-- Predict human with three models and compare with forward reach set. -->
	<node name="safety_analysis_plotter" pkg="crazyflie_human" type="safety_analysis_plotter.py" output="screen" required="true"/> 

	<!-- Read data from optitrack of human walking around. 
	<node name="human_state_estimator" pkg="crazyflie_human" type="human_state_estimator.py" output="screen"/> -->
	<node name="simulate_human$(arg human_number)" pkg="crazyflie_human" type="linear_human.py" output="screen"/> 
</launch>

